share/lib/python/neuron/rxd/rxd.py

```diff
@@ -1393,7 +1393,7 @@ def localize_index(creg, rate):
                 if ele == []:
                     mults[i] = numpy.ones(len(reg._xs))
             mults = list(itertools.chain.from_iterable(mults))
-            ics_register_reaction(0, len(all_ics_gids), len(ics_param_gids), _list_to_cint_array(all_ics_gids + ics_param_gids), numpy.asarray(mc3d_indices_start), mc3d_region_size, numpy.asarray(mults), _c_compile(fxn_string))               
+            ics_register_reaction(0, len(all_ics_gids), len(ics_param_gids), _list_to_cint_array(all_ics_gids + ics_param_gids), numpy.asarray(mc3d_indices_start, dtype=numpy.int64), mc3d_region_size, numpy.asarray(mults), _c_compile(fxn_string))               
```

Comment:
Same here, haven't touched this code

Reply:
I think I made these changes in #587 because the hybrid 1D/3D rxd tests were failing in windows. But I wasn't able to build in windows to check it worked. The default dtype `numpy.int_` is a long, which is 32bits in windows and 64bits in linux, so it's better to be specific.

Reply:
@adamjhn OK in that case we will come back to that in a separate PR once we have full testing set up on Azure CI